### ISSUE TITLE
Clarify/document nolint directive support for blocks

### ISSUE THREAD

**Issue by AlekSi** [golangci/golangci-lint#345]
For example:
```go
//nolint:golint
const (
  foo_bar = 1
  foo_baz = 2
)

var foo_quux = 3
```

It looks like it correctly disables `golint` only for `const` block. However, README does not mention this. I think it should be updated to explicitly document this (and maybe some tests should be added).

**Comment by jirfag** [comment]
hi!
yes, it disabled `golint` only for `const` block. What behavior did you expect?

**Comment by AlekSi** [comment]
I expect README to be explicit about this. Right now it says:

> If you would like to completely exclude all issues for some function prepend this comment above function:
```
//nolint
func f() {
  ...
}
```

It doesn't say it works for blocks.

**Comment by jirfag** [comment]
I've added it into [README](https://github.com/golangci/golangci-lint#nolint)
